packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -116,6 +116,7 @@ class SearchAnchor extends StatefulWidget {
     this.isFullScreen,
     this.searchController,
     this.viewBuilder,
+    this.sliverViewBuilder,
```

Comment:
Just a question, why do we want to introduce this property to `SearchAnchor`? Probably unnecessary to have two similar properties(viewBuilder and sliverViewBuilder) existing at the same time and this doesn't appear directly related to the shrinkWrap change.

Reply:
+1

Reply:
Turns out I overengineered that too